pkg/resources/warehouse.go

```diff
@@ -105,7 +106,7 @@ var warehouseSchema = map[string]*schema.Schema{
 	"statement_timeout_in_seconds": {
 		Type:        schema.TypeInt,
 		Optional:    true,
-		Default:     0,
+		Default:     172800,
```

Comment:
Default value for this parameter is `172800` (https://docs.snowflake.com/en/sql-reference/parameters.html#statement-timeout-in-seconds).  I think provider defaults should be the same as Snowflake defaults. What do you think?